ext/nnstreamer/tensor_filter/tensor_filter_python3.cc

```diff
@@ -191,12 +201,11 @@ PYCore::~PYCore ()
   gst_tensors_info_free (&inputTensorMeta);
   gst_tensors_info_free (&outputTensorMeta);
 
-  Py_LOCK ();
+  PyGILGuard gil_guard;
```

Comment:
Possible error.
With dlclose, you don't have link to Python C APIs, but your code will call PyGILRelease after dlclose.
Note that  Py_LOCK/UNLOCK does not call any Python C APIs.

Reply:
I've updated. Thank you.